Assets/Mirror/Runtime/Transport/Transport.cs

```diff
@@ -94,6 +94,14 @@ public virtual void ClientConnect(Uri uri)
 
         #region Server
 
+
+        /// <summary>
+        /// Retrieves the address of this server.
+        /// Useful for network discovery
+        /// </summary>
+        /// <returns>the url at which this server can be reached</returns>
+        public virtual Uri ServerUri() => default;
```

Comment:
this would be better in a separate pr first.
also I don't think it should be virtual. we can't assume default, so might as well make it abstract and force transport devs to implement it

Reply:
see #1454